benchmarks/README.md

```diff
@@ -0,0 +1,63 @@
+
+### Setting
+
+For this benchmark, we used the full FHVHV dataset stored in parquet files on S3. The total size of this dataset was 24.7 GiB. The Central Park Weather data was stored in a single csv file on S3 and it's total size was 514 KiB.
+
+We compared Bodo's performance on this workload to other systems including [Dask](https://www.dask.org/), [Modin on Ray](https://docs.ray.io/en/latest/ray-more-libs/modin/index.html), and [Pyspark](https://spark.apache.org/docs/latest/api/python/index.html) and observed a speedup of 20-240x. The implementations for all of these systems can be found in [`nyc_taxi`](./nyc_taxi/). Versions of the packages used are summarized below. 
+
+| Package      | Version      |
+|----------------|----------------|
+| bodo   | 2024.10   |
+| dask   | 2024.9.1  |
+| dask-cloudprovider  | 2024.9.1  |
+| modin   | 0.32.0   |
+| ray   | 2.40.0   |
+| spark  | 3.5.2 |
+
+For cluster creation and configuration, we used the [Bodo SDK](https://docs.bodo.ai/2024.12/guides/using_bodo_platform/bodo_platform_sdk_guide/) for Bodo, Dask Cloudprovider for Dask, Ray for Modin, and AWS EMR for Spark. Scripts to configure and launch clusters for each system can be found in the same directory as the implementation.
+
+Each benchmark was collected on a cluster containing 4 worker instances and 128 physical cores. Dask, Modin, and Spark used 4 `r6i.16xlarge` instances, each consisting of 32 physical cores and 256 GiB of memory. Dask Cloudprovider also allocated an additional `r6i.16xlarge` instance for the scheduler. Bodo was run on 4 `c6i.16xlarge` instances, each consisting of 32 physical cores and 64 GiB of memory.
+
+### Results
+
+The graph below summarizes the total execution time of each system (averaged over 3 runs). Results were last collected on December 12th, 2024.
+
+<img src="./img/nyc_taxi.png" alt="Monthly High Volume for Hire Vehicle Trips with Precipitation Benchmark Execution Time" title="Monthly High Volume for Hire Vehicle Trips with Precipitation Average Execution Time" width="30%">
+
+## Local Benchmark
+
+You can start to see the benefits of using Bodo from your laptop by running a smaller version of our benchmark locally. To set up, install the required packages using pip:
+
+``` shell
+pip install bodo
+pip install "dask[dataframe]"
+pip install "modin[all]"
+pip install pyspark
+```
+Then run the benchmarks!
+
+``` shell
+# (from the benchmarks/ directory)
+python -m nyc_taxi.run_local
+```
+
+We used a smaller subset of the [For Hire Vehicle High Volume dataset](https://www.nyc.gov/site/tlc/about/tlc-trip-record-data.page) to allow the workload to run locally on an Apple M2 Macbook Pro with 10 cores and 16 GB memory. Even at this smaller scale, Bodo shows a modest improvement over the next best system (Dask). 
```

Comment:
So bodo is roughly 3x better than dask or even regular pandas in this case. I wanted to highlight that for smaller data sizes these solutions can be roughly equivalent and then in the next paragraph introduce the result that shows Dask OOM which further supports what we are claiming to be good at .

Reply:
I think this is a really good result and we should showcase it as such, including a chart that we can share/ brag about

Reply:
Will open up a follow up to create the chart. Did we want to create a chart for OOM too? Also should we do a comparison with pandas here as well since it is pretty competitive on smaller datasets.